docs/signing.md

```diff
@@ -54,14 +54,17 @@ After formatting the string properly, you can convert it into bytes, for example
 import { char2Bytes } from '@taquito/utils';
 
 const bytes = char2Bytes(formattedInput);
-const payloadBytes = '05' + '0100' + char2Bytes(bytes.length) + bytes;
+const payloadBytes = '0x' + '05' + '01' + `00${char2Bytes((bytes.length / 2).toString())}` + bytes;
+
 ```
 
-The bytes representation of the string must be prefixed with 3 pieces of information:
+The bytes representation of the string must be prefixed with 5 pieces of information:
 
+- "0x" add zero-padding
```

Comment:
I think `0x` is optional in our case

Reply:
Hi @egarson just wanted to check with you if this is Klassare meant. if not i'll ping them for a review!